src/QuotePagee.js

```diff
@@ -1,6 +1,6 @@
-import { useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
-function Quote() {
+const QoutePage = () => {
```

Comment:
- [ ] I noticed that the project compilation failed due to a missing `react-router-dom` module, and there are also React Hook `useEffect` dependency warnings in the code, which should be addressed for smoother operation.
![Screenshot from 2023-09-20 11-29-03](https://github.com/MohammadYaser/math-magicians/assets/88894525/93b06c0d-88be-4a4a-a2a1-a464365cd942)